src/search/utils.go

```diff
@@ -24,9 +24,9 @@ func procBang(query string, options *engines.Options, conf *config.Config) (stri
 	query = trimBang(query)
 
 	if useSpec {
-		return query, conf.Categories[category.GENERAL].Timings, []engines.Name{specEng}
+		return query, category.GENERAL, conf.Categories[category.GENERAL].Timings, []engines.Name{specEng}
```

Comment:
Maybe this should return `category.UNDEFINED`? (Just something to think about)

Reply:
`category.UNDEFINED` should be used if an error occurs during reading the category only, but for special engine search we could add another category like `SPECENG`